app/uk/gov/hmrc/hec/services/FileCreationService.scala

```diff
@@ -166,6 +169,12 @@ class FileCreationServiceImpl @Inject() (timeProvider: TimeProvider) extends Fil
     case None                  => None
   }
 
+  private def getCTStatusMap(accountingPeriod: Option[CTAccountingPeriod]) = accountingPeriod match {
+    case Some(d: CTAccountingPeriodDigital) => ctStatusMapping(d.ctStatus.some).some
+    case Some(s: CTAccountingPeriodStride)  => ctStatusMapping(s.ctStatus).some
+    case _                                  => CTStatusMapping(None, None, Some('N')).some
```

Comment:
the `Option` in the return type is not needed here I think cos everything returns a `Some` in this function

Reply:
yes will change that one 👍